ai_for_good.qmd

```diff
@@ -51,6 +58,9 @@ Explanation: With the global population rising, sustainable and efficient farmin
 - Pest and disease detection: Early warning systems.
 - Sustainable farming practices: Optimizing water and fertilizer use.
 
+Agriculture is the common thread which holds the 17 SDGs together. Investing in the agricultural sector can address not only hunger and malnutrition but also other challenges including poverty; water and energy use; climate change; and unsustainable production and consumption. As a concrete example of TinyML being used in agriculture, researchers at UNIFEI in Brazil developed a low-cost method of detecting diseases in
```

Comment:
It would be interesting for us to mention the FarmBeat project from Microsoft, amongst others.